### DISCUSSION TITLE
The BE crashes abnormally and cannot be started.

### DISCUSSION THREAD

**Discussion by donglin94** [apache/doris#8468] [A - General / Q&A] [answered]
**问题描述：**
BE 异常崩溃，无法启动, 只有少量任务实时写入Doris
`start_be.sh: line 102:  7898 Segmentation fault      $LIMIT ${DORIS_HOME}/lib/palo_be "$@" >> $LOG_DIR/be.out 2>&1 < /dev/null`
**问题复现的case：**
   
**Doris版本：**
palo-0.15.1-rc09
**Doris集群基本信息：**
fe 1台 16c64g
be 13台 16c64g
**异常信息：**
```
start time: Mon Mar 14 17:58:31 CST 2022
*** Aborted at 1647251912 (unix time) try "date -d @1647251912" if you are using GNU date ***
PC: @          0x1684a63 doris::memory_copy()
*** SIGSEGV (@0x0) received by PID 22945 (TID 0x7fe6ead4c700) from PID 0; stack trace: ***
    @          0x3022d82 google::(anonymous namespace)::FailureSignalHandler()
    @     0x7fe746090630 (unknown)
    @          0x1684a63 doris::memory_copy()
    @          0x1684dac doris::FieldTypeTraits<>::direct_copy()
    @          0x16c63d7 doris::Reader::_agg_key_next_row()
    @          0x16bbcd8 doris::Merger::merge_rowsets()
    @          0x16ac47b doris::Compaction::do_compaction_impl()
    @          0x16adc45 doris::Compaction::do_compaction()
    @          0x16aed8a doris::CumulativeCompaction::execute_compact_impl()
    @          0x16a9b13 doris::Compaction::execute_compact()
    @          0x1635278 doris::Tablet::execute_compaction()
    @          0x160846a _ZNSt17_Function_handlerIFvvEZN5doris13StorageEngine35_compaction_tasks_producer_callbackEvEUlvE0_E9_M_invokeERKSt9_Any_data
    @          0x198a9e3 doris::ThreadPool::dispatch_thread()
    @          0x1984b2c doris::Thread::supervise_thread()
    @     0x7fe746088ea5 start_thread
    @     0x7fe745aaf9fd __clone
    @                0x0 (unknown)

```

**Comment by dataalive** [comment, marked as answer]
fixed pr : [https://github.com/apache/incubator-doris/pull/8515](url)

**Comment by haoranchuixue** [comment]
2.x版本写入数据超过内存大小就会导致BE挂掉，而且再无法启动。只有如下警告
I1026 18:53:46.751749 23846 daemon.cpp:218] OS physical memory 61.76 GB. Process memory usage 20.65 GB, limit 49.41 GB, soft limit 44.46 GB. Sys available memory 39.14 GB, low water mark 1.60 GB, warning water mark 3.20 GB. Refresh interval memory 
growth 0 B

**Reply by zhbdesign** [reply]
看看日志，是不是有副本均衡和合并任务，可以全部关掉后，再启动

**Comment by Zhang21** [comment]
use jdk1.8

ref: <https://github.com/apache/doris/issues/22816#issuecomment-1678334822>

**Comment by Zhang21** [comment]
please attention the version of jdk。 `start_(be|fe).sh` contains the version of jdk!

![image](https://github.com/apache/doris/assets/18358275/be040127-2732-48a0-879e-e2c71c8e7cac)
